Jenkinsfile.release

```diff
@@ -74,85 +78,95 @@ podTemplate(cloud: 'openshift', label: pod_label, yaml: pod) {
+            cmd-buildprep --build=${params.VERSION} \
+                --arch=all s3://${s3_stream_dir}/builds
             """)
+        }
 
-            def basearch = shwrapCapture("cosa basearch")
+        for (basearch in params.ARCH.split()) {
             def meta_json = "builds/${params.VERSION}/${basearch}/meta.json"
             def meta = readJSON file: meta_json
-            if (meta.gcp?.image) {
-                gcp_image = meta.gcp.image
-            }
-        }
 
-        // For production streams, import the OSTree into the prod
-        // OSTree repo.
-        if ((params.STREAM in streams.production) && utils.pathExists("/etc/fedora-messaging-cfg/fedmsg.toml")) {
-            stage("OSTree Import: Prod Repo") {
-                shwrap("""
-                /var/tmp/fcos-releng/coreos-ostree-importer/send-ostree-import-request.py \
-                    --build=${params.VERSION} --s3=${s3_stream_dir} --repo=prod \
-                    --fedmsg-conf=/etc/fedora-messaging-cfg/fedmsg.toml
-                """)
+            // For production streams, import the OSTree into the prod
+            // OSTree repo.
+            if ((params.STREAM in streams.production) && utils.pathExists("/etc/fedora-messaging-cfg/fedmsg.toml")) {
+                stage("OSTree Import ${basearch}: Prod Repo") {
+                    shwrap("""
+                    /var/tmp/fcos-releng/coreos-ostree-importer/send-ostree-import-request.py \
+                        --build=${params.VERSION} --arch=${basearch} \
+                        --s3=${s3_stream_dir} --repo=prod \
+                        --fedmsg-conf=/etc/fedora-messaging-cfg/fedmsg.toml
+                    """)
+                }
             }
-        }
 
-        // For production streams, promote the GCP image so that it
-        // will be the chosen image in an image family and deprecate
-        // all others. `ore gcloud promote-image` does this for us.
-        if (params.STREAM in streams.production) {
-            stage('GCP: Image Promotion') {
-                shwrap("""
-                # pick up the project to use from the config
-                gcp_project=\$(jq -r .project_id \${GCP_IMAGE_UPLOAD_CONFIG})
-                ore gcloud promote-image \
-                    --log-level=INFO \
-                    --project=\${gcp_project} \
-                    --json-key \${GCP_IMAGE_UPLOAD_CONFIG} \
-                    --family fedora-coreos-${params.STREAM} \
-                    --image "${gcp_image}"
-                """)
+            // For production streams, promote the GCP image so that it
+            // will be the chosen image in an image family and deprecate
+            // all others. `ore gcloud promote-image` does this for us.
+            if ((basearch == 'x86_64') && params.STREAM in streams.production) {
+                stage('GCP ${basearch}: Image Promotion') {
+                    if (meta.gcp?.image) {
+                        gcp_image = meta.gcp.image
```

Comment:
Minor/optional: and also feels like we can just nuke that variable and use `meta.gcp.image` directly in the invocation?

Reply:
ok tried to modify it to be better